Makefile

```diff
@@ -33,7 +33,7 @@ bazel-push-images:
 push: bazel-push-images
 
 bazel-test:
-	hack/dockerized "hack/bazel-fmt.sh && CI=${CI} ARTIFACTS=${ARTIFACTS} hack/bazel-test.sh"
+	hack/dockerized "hack/bazel-fmt.sh && CI=${CI} ARTIFACTS=${ARTIFACTS} WHAT=${WHAT}  hack/bazel-test.sh"
```

Comment:
A convention is to use `KUBEVIRT_`, maybe `KUBEVIRT_UNIT_TESTS`?

Reply:
@xpivarc `WHAT` is already in the code and I have followed this [review](https://github.com/kubevirt/kubevirt/pull/8361#discussion_r966804843)